src/app/view/components/dot-device-selector/dot-device-selector.component.spec.ts

```diff
@@ -71,12 +72,58 @@ describe('DotDeviceSelectorComponent', () => {
         const devicesMock = mockDotDevices.filter(
             (device: DotDevice) => +device.cssHeight > 0 && +device.cssWidth > 0
         );
-        expect(component.options.length).toEqual(devicesMock.length + 1);
+        expect(component.options.length).toEqual(2);
+        expect(component.options[0]).toEqual(defaultDevice);
+        expect(component.options[1]).toEqual(devicesMock[0]);
     });
 
     it('shoudl set fixed width to dropdown', () => {
         fixture.detectChanges();
         const pDropDown: Dropdown = de.query(By.css('p-dropdown')).componentInstance;
         expect(pDropDown.style).toEqual({ width: '100px' });
     });
+
+});
+
+
+describe('DotDeviceSelectorComponent', () => {
```

Comment:
Two describes with the exact same name is not a good approach, wrap this in one describe: `DotDeviceSelectorComponent` and create 2 more for this separation.